docker/.dockerignore

```diff
@@ -0,0 +1,3 @@
+/*.yml
+/localsettings.py
+/Dockerfile_commcarehq_base
```

Comment:
Is this dockerfile (path had been /docker/dockerfiles/Dockerfile_commcarehq_base) used to build [dimagi/commcarehq_base](https://hub.docker.com/r/dimagi/commcarehq_base/) on docker hub? It appears that image is build automatically on a schedule; is that correct? Do I need to change anything if I've moved the file within this repo?